db/schema.rb

```diff
@@ -19,10 +19,10 @@
     t.string "attachment"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "enrollment_id"
     t.string "type"
     t.boolean "archive", default: false
-    t.integer "dgfip_id"
+    t.string "attachable_type"
+    t.integer "attachable_id"
```

Comment:
after reading the whole PR, id do not get what the attachable is ? Is this the document or the enrollment it's attached to ?

Reply:
it is a Rails trick attachable refers to documents -> http://guides.rubyonrails.org/association_basics.html#polymorphic-associations